packages/cli/docs-importers/mintlify/src/MintlifyImporter.ts

```diff
@@ -95,6 +95,13 @@ export class MintlifyImporter extends DocsImporter<MintlifyImporter.Args> {
             }
         }
         this.context.logger.debug("Imported OpenAPI specs");
+
+        // We're currently setting one instance so that clients can generate docs out of the box,
+        // but they'll have the option to add additional instances and custom domains later.
+        builder.setInstance({
+            companyName: mint.name
+        });
+        this.context.logger.debug("Set instance");
```

Comment:
do we want to output the name here, as well?

Reply:
Good idea, updating now!

Reply:
@RohinBhargava are you thinking that we should output the name from the mint file or the name we auto-generate?

Reply:
good question! I think you have the most context, but I would assume `mint.name`, given the locality of the debug statement to the invocation

Reply:
@mattblank11 could see this being:  `Added instance ${url} to docs.yml`

Reply:
Appreciate the input!

I agree with @dsinghvi on returning the url, if anything. IMO it's weird to return a string from the `setInstance` function but I don't love the alternative of creating a separate `getInstanceUrl` function. Feel free to push back on this!